data.json

```diff
@@ -550,5 +550,11 @@
     "githubUsername": "Ibtihel-ouni",
     "imageUrl": "https://user-images.githubusercontent.com/58151963/107116883-97f86480-6876-11eb-8382-fc1c6ce06403.png",
     "issueNumber": 338
+  },
+  {
+    "name": "Vaidhyanathan S M",
+    "githubUsername": "smv1999",
+    "imageUrl": "https://drive.google.com/file/d/1YaiFeRqbPNzxNON3UjtPXaqiCmuNErXD/view?usp=sharing",
```

Comment:
Please paste your image into your issue message. It will give you a link to put in here 👍